### PR TITLE
Remove responders gem

### PR DESCRIPTION
**Pull request by arunagw**
* We can set defaults in routes so we can reject other requests from routes
level
* `responders` is not a core rails part

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,16 +9,16 @@
 
   resource :account, only: [:show]
   resources :builds, only: [:create]
-  resource :credit_card, only: [:update]
+  resource :credit_card, only: [:update], defaults: { format: :json }
```
Comment (srt32) [inline config/routes.rb]: Would there be a non-namespace breaking way to wrap these json endpoints into a block with a single call to `defaults: { format: :json }`?

Comment (Koronen) [inline config/routes.rb]: [`Object#with_options`](http://apidock.com/rails/Object/with_options) maybe?

**Comment by gylaz** [comment]
I thought responders where the recommended way with Rails. Did that change? pinging @sgrif

**Comment by sgrif** [comment]
There's no official recommendation either way, we just moved `respond_with` into the responders gem because DHH doesn't like the feature and didn't want it in Rails.

**Comment by salbertson** [comment]
Looks good. :+1:

File: config/routes.rb [hunk 2]
```diff
@@ -9,16 +9,20 @@
 
   resource :account, only: [:show]
   resources :builds, only: [:create]
-  resource :credit_card, only: [:update]
 
   resources :repos, only: [:index] do
-    resource :activation, only: [:create]
-    resource :deactivation, only: [:create]
-    resource :subscription, only: [:create, :destroy]
+    with_options( defaults: { format: :json } ) do
```
Comment (teoljungberg) [inline config/routes.rb]: Remove the spaces around `(`

**Pull request merged**